client/components/CategoryList.jsx

```diff
@@ -4,15 +4,15 @@ import Category from './Category.jsx';
 const CategoryList = (props) => {
   return (
     <div id="nav-category-list">
-      <Category name="Cat 1"/>
-      <Category name="Cat 2"/>
-      <Category name="Cat 3"/>
-      <Category name="Cat 4"/>
-      <Category name="Cat 5"/>
-      <Category name="Cat 6"/>
-      <Category name="Cat 7"/>
-      <Category name="Cat 8"/>
-      <Category name="Cat 9"/>
+      <Category name="Wands"/>
+      <Category name="Quidditch"/>
+      <Category name="Spells"/>
+      <Category name="Accessories"/>
+      <Category name="Rarities"/>
+      <Category name="Food"/>
+      <Category name="Furniture"/>
+      <Category name="Lamps"/>
+      <Category name="Lord of The Rings"/>

```

Comment:
Mapping through an array of categories in state would make this look nicer. Your call!

Reply:
yes it is planned!